charts/fybrik/values.yaml

```diff
@@ -479,6 +479,106 @@ katalogConnector:
 
   affinity: {}
 
+# OpenMetadata connector component
+openmetadataConnector:
+  # Set to true to deploy the openmetadata connector or false to skip its deployment.
+  # Defaults to true if `coordinator.catalog` is set to "openmetadata"
+  enabled: auto
+
+  # Image name or a hub/image[:tag]
+  image: "ghcr.io/cdoron/openmetadata-connector:0.0.0"
```

Comment:
cdoron -> fybrik

Reply:
Currently the image resides under the cdoron namespace.
This will change in the future

Reply:
this file is removed. Thanks